src/main/java/com/scalar/dl/client/contract/ValidateLedger.java

```diff
@@ -46,14 +46,14 @@ public JsonObject invoke(Ledger ledger, JsonObject argument, Optional<JsonObject
             .withStartAge(startAge, true)
             .withEndAge(endAge, true)
             .withAgeOrder(AgeOrder.ASC);
-    List<Asset> assets = ledger.scan(filter);
+    List<Asset<JsonNode>> assets = ledger.scan(filter);
+
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
You can just call `getObjectMapper()` to get an ObjectMapper instance so that you can re-use it. 
https://github.com/scalar-labs/scalardl/blob/master/docs/how-to-write-contract.md#write-a-simple-contract